### PR TITLE
WRO-2188: [React18] Apply `createRoot` API to test app

### PR DESCRIPTION
**Pull request by hong6316**
### Issue Resolved / Feature Added
In this PR, apply React18's createRoot API to test app

### Resolution
update react 18 API(createRoot) on index.js

### Additional Considerations
fix linting warnings

### Links
WRO-2188

### Comments
Enact-DCO-1.0-Signed-off-by: Taeyoung Hong (taeyoung.hong@lge.com)

### CODE REVIEW

File: screenshot/index.js [hunk 1]
```diff
@@ -18,13 +18,9 @@ if ('testId' in props) props.testId = Number.parseInt(props.testId);
 if ('request' in props) {
 	window.__TEST_DATA = testMetadata;
 } else if (Object.keys(props).length) {
-	render(
-		<App {...props} />,
-		document.getElementById('root')
-	);
+	createRoot(document.getElementById('root'))
+	.render(<App {...props} />);
```
Comment (MikyungKim) [inline screenshot/index.js:22]: We have lint warnings. Please fix them.

Comment (hong6316) [reply screenshot/index.js]: OK

File: .travis.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-    - "node"
+    - "14"
```
Comment (MikyungKim) [inline .travis.yml:3]: Why is this needed?

Comment (hong6316) [reply .travis.yml:3]: It occurs a travis error while installing node.
https://app.travis-ci.com/github/enactjs/ui-test-utils/builds/250040528
> $ nvm install node
node: /lib/x86_64-linux-gnu/libm.so.6: version 'GLIBC_2.27' not found (required by node)
node: /lib/x86_64-linux-gnu/libc.so.6: version 'GLIBC_2.25' not found (required by node)
node: /lib/x86_64-linux-gnu/libc.so.6: version 'GLIBC_2.28' not found (required by node)
The command "npm config set spin false" failed and exited with 1 during .
Your build has been stopped.

Comment (MikyungKim) [reply .travis.yml:3]: Please refer to https://github.com/enactjs/sandstone/blob/develop/.travis.yml#L9
and update it.

**Review by MikyungKim — APPROVED** [review]
LGTM

File: .travis.yml [hunk 3]
```diff
@@ -4,7 +4,7 @@ node_js:
     - "node"
 before_install:
     - sudo apt-get update
-    - sudo apt-get install build-essential libcairo2-dev libpango1.0-dev libjpeg-dev libgif-dev librsvg2-dev
+    - sudo apt-get install build-essential libcairo2-dev libpango1.0-dev libjpeg-dev libgif-dev librsvg2-dev
```
Comment (MikyungKim) [inline .travis.yml:7]: Please remove the empty space.
